ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -444,7 +444,7 @@ default List<VirtualColumn> acidVirtualColumns() {
   }
 
   /**
-   * {@link org.apache.hadoop.hive.ql.parse.UpdateDeleteSemanticAnalyzer} rewrites DELETE/UPDATE queries into INSERT
+   * {@link UpdateSemanticAnalyzer} rewrites DELETE/UPDATE queries into INSERT
```

Comment:
same, check javadoc refs for UpdateSemanticAnalyzer and mention Delete as well

Reply:
Updated.